Source/WebKit/WebProcess/WebProcess.h

```diff
@@ -47,6 +47,7 @@
 #include <WebCore/PageIdentifier.h>
 #include <WebCore/PluginData.h>
 #include <WebCore/RegistrableDomain.h>
+#include <WebCore/RenderTreeAsText.h>
```

Comment:
Can the enum be forward declared instead of adding a new include to a header and slowing down the build?